src/web/components/StickyBottomBanner/BrazeBanner.tsx

```diff
@@ -186,7 +216,14 @@ export const canShow = async (
     }
 
     try {
-        return getMessageFromBraze(apiKey as string, brazeUuid);
```

Comment:
Since we're in an `async` function here we could also use `await` if we felt it would make things more readable. This would look like:

```js
const result = await getMessageFromBraze(apiKey as string, brazeUuid);

const timeTaken = canShowTiming.endAndReturn();
record({
  component: 'braze-banner-timing',
  value: timeTaken,
});

return result;

```

I think I have a _slight_ preference for this, as it's also more consistent with the rest of this file. But I definitely don't feel strongly either way!

Reply:
I like it! Will work well in Frontend too.